micrometer-core/src/main/java/io/micrometer/core/aop/TimedAspect.java

```diff
@@ -92,16 +104,46 @@ public Object timedMethod(ProceedingJoinPoint pjp) throws Throwable {
         } finally {
             try {
                 sample.stop(Timer.builder(metricName)
-                        .description(timed.description().isEmpty() ? null : timed.description())
-                        .tags(timed.extraTags())
-                        .tags(EXCEPTION_TAG, exceptionClass)
-                        .tags(tagsBasedOnJoinPoint.apply(pjp))
-                        .publishPercentileHistogram(timed.histogram())
-                        .publishPercentiles(timed.percentiles().length == 0 ? null : timed.percentiles())
-                        .register(registry));
+                                    .description(timed.description().isEmpty() ? null : timed.description())
+                                    .tags(timed.extraTags())
+                                    .tags(EXCEPTION_TAG, exceptionClass)
```

Comment:
It looks like the exception tag isn't applied to the LongTaskTimer. I can add this post-merge, though, unless there was some reason it wasn't included.

Reply:
Nevermind. I see now why we can't tag the exception class.